apps/src/utils.js

```diff
@@ -431,3 +433,22 @@ exports.ellipsify = function (inputText, maxLength) {
   }
   return inputText || '';
 };
+
+/**
+ * Extends _.merge to concat rather than overwrite values in arrays.
+ *
+ * @see https://lodash.com/docs#mergeWith
+ *
+ * @param {Object} object
+ * @param {Object} other
+ * @returns {Object} original object (now modified in-place)
+ */
+exports.mergeConcatArrays = (object, other) => {
+  const mergeCustomizer = (objValue, srcValue) => {
+    if (_.isArray(objValue)) {
+      return objValue.concat(srcValue);
+    }
+  };
+
+  return _.mergeWith(object, other, mergeCustomizer);
+};
```

Comment:
I wonder if things would be easier to reason about if this helper returned a new object (instead of mutating the passed in object). The caller can then still make a decision about whether it overwrites the original, but it makes it a little more explicit.